x-pack/test/functional_embedded/config.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resolve } from 'path';
+// import { CA_CERT_PATH, KBN_CERT_PATH, KBN_KEY_PATH } from '@kbn/dev-utils';
+import { FtrConfigProviderContext } from '@kbn/test/types/ftr';
+import { pageObjects } from '../functional/page_objects';
+
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const kibanaFunctionalConfig = await readConfigFile(require.resolve('../functional/config.js'));
+
+  const iframeEmbeddedPlugin = resolve(__dirname, './plugins/iframe_embedded');
+
+  const servers = {
+    ...kibanaFunctionalConfig.get('servers'),
+    elasticsearch: {
+      ...kibanaFunctionalConfig.get('servers.elasticsearch'),
+    },
+    kibana: {
+      ...kibanaFunctionalConfig.get('servers.kibana'),
+      // protocol: 'https',
```

Comment:
@elastic/kibana-qa I need to setup TLS to test https://github.com/elastic/kibana/pull/68108 properly. What I should do to make WebDriver to work with certs from `@kbn/dev-utils`? When I run FTR locally I can see errors in both Chrome and Firefox
![2020-06-08_16-19-30](https://user-images.githubusercontent.com/3198181/84061435-323d9b00-a9be-11ea-8518-7ccae6ae3bcb.png) 
setting env var `NODE_TLS_REJECT_UNAUTHORIZED="0"` doesn't help.

Reply:
Try doing the both: 
```TEST_IGNORE_CERT_ERRORS=1``` and 
``` NODE_TLS_REJECT_UNAUTHORIZED=0```

Reply:
@liza-mae ty! will try tomorrow. Is it possible to specify env vars via FTR config?

Reply:
Nope, it didn't work out. I cannot find any mentions of `TEST_IGNORE_CERT_ERRORS` in Kibana repo either.

Reply:
What error are you getting?  Seems like CI passed.

Reply:
@restrry - sorry I missed seeing the error above, this error comes in the server logs.  If so, then what I sent you will not fix that, what I sent you is just for the FTR to ignore the certificate errors when you don't have TLS setup.  In your case you have setup TLS and want to pass in the certificates.  Maybe use this as an example: ```test/server_integration/http/ssl/config.js``` 
Looks like you can pass in the certs: 
```
serverArgs: [
        ...httpConfig.get('kbnTestServer.serverArgs'),
        '--server.ssl.enabled=true',
        `--server.ssl.key=${KBN_KEY_PATH}`,
        `--server.ssl.certificate=${KBN_CERT_PATH}`,
```
and I would check with Kibana Ops team to confirm.

Reply:
As I can see this error pops up when I run FTR and it calls `/status` endpoint
```
 Error: 4550243776:error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../deps/openssl/openssl/ssl/record/rec_layer_s3.c:1550:SSL alert number 46
```
I suspect additional setup on the FTR side required.